### PR TITLE
Add an info message in `neuro run`/`neuro submit`

### PR DESCRIPTION
**Pull request by ayushkovskiy**
Closes https://github.com/neuromation/platform-client-python/issues/943

### CODE REVIEW

File: neuromation/cli/job.py [hunk 1]
```diff
@@ -853,6 +853,12 @@ def format_fail(job: str, reason: Exception) -> str:
         await browse_job(root, job)
 
     if not detach:
+        click.echo(
+            "This terminal is attached to the remote job, so you receive "
+            "the job output.\nUse 'Ctrl-C' to detach from this job (it "
```
Comment (serhiy-storchaka) [inline neuromation/cli/job.py]: Could it be reformatted so that lines will break after `\n`?

Comment (ayushkovskiy) [reply neuromation/cli/job.py]: done, thanks

**Comment by asvetlov** [comment]
@ayushkovskiy what do you think about my suggestion for gray color (`dim=True`)?

**Comment by ayushkovskiy** [comment]
> what do you think

@asvetlov just pushed :wink:

**Pull request merged**